migrations/Version20220519131327.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220519131327 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE book_category (book_id UUID NOT NULL, category_id UUID NOT NULL, PRIMARY KEY(book_id, category_id))');
+        $this->addSql('CREATE INDEX IDX_1FB30F9816A2B381 ON book_category (book_id)');
+        $this->addSql('CREATE INDEX IDX_1FB30F9812469DE2 ON book_category (category_id)');
+        $this->addSql('COMMENT ON COLUMN book_category.book_id IS \'(DC2Type:uuid)\'');
+        $this->addSql('COMMENT ON COLUMN book_category.category_id IS \'(DC2Type:uuid)\'');
+        $this->addSql('CREATE TABLE category (id UUID NOT NULL, name VARCHAR(255) NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('COMMENT ON COLUMN category.id IS \'(DC2Type:uuid)\'');
+        $this->addSql('ALTER TABLE book_category ADD CONSTRAINT FK_1FB30F9816A2B381 FOREIGN KEY (book_id) REFERENCES book (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE book_category ADD CONSTRAINT FK_1FB30F9812469DE2 FOREIGN KEY (category_id) REFERENCES category (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE book DROP category');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
```

Comment:
```suggestion
```

I would even prevent to go down method by calling `$this->throwIrreversibleMigrationException()`